examples/bulk_base_api.cxx

```diff
@@ -341,9 +341,9 @@ main()
 
     auto options = couchbase::cluster_options(arguments.username, arguments.password);
     options.apply_profile("wan_development");
-    auto [cluster, ec] = couchbase::cluster::connect(io, arguments.connection_string, options).get();
-    if (ec) {
-        fmt::print("Unable to connect to cluster at \"{}\", error: {}\n", arguments.connection_string, ec.message());
+    auto [connect_err, cluster] = couchbase::cluster::connect(io, arguments.connection_string, options).get();
+    if (connect_err) {
+        fmt::print("Unable to connect to cluster at \"{}\", error: {}\n", arguments.connection_string, connect_err.message());
```

Comment:
We should just print the whole `connect_err` here, since we defined the serialization in fmt/error.hxx.

Reply:
```suggestion
        fmt::print("Unable to connect to cluster at \"{}\", error: {}\n", arguments.connection_string, connect_err);
```